geo_concerns/app/models/image.rb

```diff
@@ -0,0 +1,7 @@
+# Base class for Images that are georeferenced with a bbox
+class Image < ActiveFedora::Base
+  include ::GeoConcerns::ImageBehavior
+  validates_presence_of :title,  message: 'Your work must have a title.'
+  validates_presence_of :georss_box,  message: 'Your work must have a bbox.'
```

Comment:
For Image, it'd be width and height, right? Then all geo classes would be title, bbox. For Vector/Raster we'd also include crs. I'm assuming that the `validates_presence_of` is for the properties(?)